File: January 21, 2026.py
def parse_inline_code(markdown):

    new_markdown = ""
    close_tag = False

    for char in markdown:
        if char == "`":
            if close_tag:
                new_markdown += "</code>"
            else:
                new_markdown += "<code>"
            close_tag = not close_tag
        else:
            new_markdown += char

    return new_markdown

File: January 21, 2026.js
function parseInlineCode(markdown) {

  let newMarkdown = "";
  let close_tag = false;

  for (let char of markdown) {
    if (char == "`") {
      if (close_tag) {
        newMarkdown += "</code>"
      } else {
        newMarkdown += "<code>"
      }
      close_tag = !close_tag
    } else {
      newMarkdown += char
    }
  }

  return newMarkdown;
}